openassessment/templates/openassessmentblock/edit/oa_training_example.html

```diff
@@ -3,9 +3,7 @@
 <li class="openassessment_training_example is-collapsible">
 
     <div class="openassessment_training_example_header view-outline">
-        <a class="action expand-collapse collapse">
```

Comment:
Note to reviewers: this icon missed the fa conversion. When I changed it the icon did appear, but the functionality did not work (the section did not collapse when the icon or title beside it was clicked on). I created a bug to fix this if product desires, and am removing the icon so it does not look confusing.

https://openedx.atlassian.net/browse/TNL-3799